provider/bitmovin/bitmovin.go

```diff
@@ -987,28 +992,83 @@ func (p *bitmovinProvider) addOutputStatusInfo(job *db.Job, status *provider.Job
 	return nil
 }
 
-func (p *bitmovinProvider) addSourceInfo(job *db.Job, status *provider.JobStatus) error {
-	encodingS := services.NewEncodingService(p.client)
-	resp, err := encodingS.ListStream(job.ProviderJobID, 0, 1)
+func (p *bitmovinProvider) addOutputFilesInfo(job *db.Job, status *provider.JobStatus) error {
+	// TODO: we only have info for MP4. Need to add for webm and mov.
+
+	muxings, err := p.listMP4Muxing(job.ProviderJobID)
 	if err != nil {
 		return err
 	}
-	if len(resp.Data.Result.Items) > 0 {
-		streamID := stringValue(resp.Data.Result.Items[0].ID)
-		streamInput, err := encodingS.RetrieveStreamInputData(job.ProviderJobID, streamID)
+
+	var files []provider.OutputFile
+	for _, muxing := range muxings {
+		encodingS := services.NewEncodingService(p.client)
```

Comment:
Should we create ``encodingS`` outside the forloop?

Reply:
Good catch!   It's a copy-paste error.